org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/util/widget/GradleProjectSettingsComposite.java

```diff
@@ -109,6 +116,13 @@ private void createBuildScansCheckbox(Composite parent) {
         HoverText.createAndAttach(this.buildScansCheckbox, CoreMessages.Preference_Label_BuildScansHover);
     }
 
+    private void createAutoBuildCheckbox(Composite parent) {
+        this.autoBuildCheckbox = new Button(parent, SWT.CHECK);
+        this.autoBuildCheckbox.setText("Automatic Refresh");
+        GridDataFactory.swtDefaults().align(SWT.FILL, SWT.CENTER).grab(true, false).span(2, 1).applyTo(this.autoBuildCheckbox);
+        HoverText.createAndAttach(this.autoBuildCheckbox, "Automatically synchronize project when build script changes");
```

Comment:
Change text to `Automatically start project synchronization in the background when the build script changes`